code/game/gamemodes/events/powercreeper.dm

```diff
@@ -137,7 +137,11 @@
 		return 0
 	Beam(M, "lighting", 'icons/obj/zap.dmi', 5, 2)
 	playsound(src,'sound/weapons/electriczap.ogg',50, 1) //we still want a sound
-	return electrocute_mob(M, powernet, src)
+	if(!electrocute_mob(M, powernet, src))
+		var/damage = (10/(-1*(powernet.avail / 5000)))+45
+		if(damage > 0) //dont wanna heal em, damage only starts at around ~1kW
```

Comment:
Could add a min to the damage var above so that it can't go lower than 0

Reply:
how wat

Reply:
You can skip it from having to check if damage is above 0 if you add a min(0, [that long string of calculations]) to the damage var above

Reply:
nvm